app/posts/personalised-prevention-services/2025/06/2025-06-03-definitions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Personalised prevention: working definitions"
+description: "Working definitions of terms we use when we’re talking about personalised prevention"
+date: 2025-06-03
+---
+
+These working definitions aim to give people working on personalised prevention services a shared understanding of the language they’re using internally.
+
+## Prevention
+
+Any work or activity we do to stop someone getting ill with a named clinical condition.
+
+## Clinical versus lifestyle
```

Comment:
Is it worth saying something about this being an over simplification to split by "Clinical" and "Lifestyle" – for example some weight management services are supported by clinicians like a dietitian-led service for weight management, or could a clinician work on a service that supports behaviour change (like gardening), sorry if I'm splitting hairs.

Maybe its the title "Clinical treatment" and "Lifestyle and behaviour change serivces"